bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/guide/BoundaryCheck.java

```diff
@@ -0,0 +1,6 @@
+package edu.gemini.spModel.guide;
+/**
+   * Enumeration of possible results when checking boundaries.
+   */
+  public enum BoundaryCheck{inRange, nearBoundary, outOfRange, vignetting}
```

Comment:
This should be specific to PWFS since it is the only one with a vignetting boundary.  I would put it in the same package as the `PwfsGuideProbe` and call it something like `PwfsProbeRangeArea` to tie it to PWFS.  Maybe we only include `nearBoundary` if we're going to distinguish that region for a distinct warning.

Reply:
I'll remove `nearBoundary`, it's coming from my misunderstanding of how to handle vignetting.